examples/dodge-the-creeps/rust/src/main_scene.rs

```diff
@@ -78,7 +77,7 @@ impl Main {
             .base
             .get_node_as::<PathFollow2D>("MobPath/MobSpawnLocation");
 
-        let mut mob_scene: Gd<RigidBody2D> = instantiate_scene(&self.mob_scene);
+        let mut mob_scene: Gd<RigidBody2D> = self.mob_scene.instantiate_as();
```

Comment:
Maybe use this notation, as it reads more idiomatically "instantiate as RigidBody2D":
```suggestion
        let mut mob_scene = self.mob_scene.instantiate_as::<RigidBody2D>();
```